insights/parsers/getconf.py

```diff
@@ -0,0 +1,21 @@
+"""
+getconf PAGE_SIZE - command ``/usr/sbin/getconf PAGE_SIZE``
+=============================================
+
+This very simple parser returns the output of the ``getconf PAGE_SIZE`` command.
+
+    >>> output = shared[getconfpagesize]
+    >>> output['value']
+    '4096'
+"""
+from .. import parser
+from insights.specs import Specs
+
+
+@parser(Specs.getconf_page_size)
+def getconfpagesize(context):
```

Comment:
Better refactor this Parser method to a `Class` and provide an attribute e.g. `value`. (e.g. https://github.com/RedHatInsights/insights-core/blob/master/insights/parsers/hostname.py)